resources/assets/js/components/BlockOptions.vue

```diff
@@ -18,53 +18,60 @@
 				v-for="field in currentDefinition.fields"
 				:label="field.label"
 				:prop="field.name"
-				:error="localErrors[field.name] || null"
+				:error="
```

Comment:
If the errors are nested we set them to null in the parent field "wrapper" (collection/group) and instead pull them in from the collection and group component.